### ISSUE TITLE
Noisy notification due to custom keyword not visible 

### ISSUE THREAD

**Issue by kianenigma** [SchildiChat/schildichat-desktop#174]
Assume I get a notifications in a room, and the notification is _noisy_ and turn red in the channel list. 

when I go into the room, Element renders the message that caused the noisy notification as red entirely but SchildiChat does not. This is perhaps the only thing that I miss from Element. 

for @mentions this is okay, but if a notification is supposed to be noisy due to a custom keyword, then it is near impossible to find it.

**Comment by kianenigma** [comment]
IN fact, neither Element on Schildi do the right thing: the keyword should be highlighted, not the entire message :)

**Comment by su-ex** [comment]
Right, it makes far more sense to highlight/pillify the keywords only, but I guess this is more difficult to implement.
For highlighting the entire message, I refer to this: https://github.com/SchildiChat/schildichat-desktop/issues/141
Sadly, I've currently no time to implement either, so contributions only / not anytime soon™ probably.
Since highlighted keywords / pills in general would probably have broader acceptance, this would be the preferred way to strive for for schildi. Since that other issue already exists, let's have this one for highlighted/pillified keywords.

**Comment by su-ex** [comment]
Related: https://github.com/SchildiChat/schildichat-desktop/issues/141
